### ISSUE TITLE
Trying to compile in Ubuntu and I still get the same error

### ISSUE THREAD

**Issue by SuperX-dev** [ndless-nspire/Ndless#268]
Hi,

Recently I installed the Ndless-SDK in Ubuntu running in VirtualBox. Once I installed it, I tried to compile the HelloWorld test script in "~/Ndless/ndless-sdk/samples/helloworld-sdl" and when I execute the `make` command, I get this in the terminal:

```
nspire-gcc -Wall -W -marm -Os -c main.c -o main.o
mkdir -p .
nspire-ld main.o -o test.elf 
arm-none-eabi-ld: cannot find -lnspireio
arm-none-eabi-ld: cannot find -lndls
arm-none-eabi-ld: cannot find -lsyscalls
collect2: error: ld returned 1 exit status
make: * [Makefile:40: test.elf] Error 1
```

Do you know what is this error I´m getting? I tried to search for it but I got really confused.
Sorry if I sound like a Real Noob.

**Comment by Vogtinator** [comment]
That means you didn't run `make` in the root directory of the repository.

**Comment by SuperX-dev** [comment]
Sorry, but I´m not understanding what are you trying to say with the "root directory of the repository".

What I do on the terminal to run `make`:

![imagem](https://user-images.githubusercontent.com/74361788/102015417-69b54780-3d53-11eb-9b12-354224884fab.png)

After I run `make`, it only creates this "hello-sdl.o" file and it has around 1.5KB

**Comment by adriweb** [comment]
Run "make" inside the "ndless-sdk" folder of your local clone of the ndless repo.

**Comment by Vogtinator** [comment]
https://hackspire.org/index.php/C_and_assembly_development_introduction states:

> Build Ndless and the SDK, in the top level of the repository, run `make`

So in your case, inside the `~/Ndless` directory. Though inside `ndless-sdk` also works in this case.

**Comment by SuperX-dev** [comment]
Ok. I copy and pasted the folder into the `~/Ndless/ndless-sdk` then ran `make` and it worked.

But not on first try:

I was getting an error message, similar to the #238 issue thread. 
So, with that, I have decided to run `build_toolchain.sh` and it said something like "...(python3/3-dev) dependencies are missing." With that, I googled how I could install python3 and pip. After that I run `build_toolchain.sh` again and then `make`. 
And now it´s working.
Thank you for your support.
